CHANGELOG.md

```diff
@@ -1,19 +1,5 @@
 # Change Log - AWS SDK for Android
 
-## [Release 2.16.13](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.16.13)
```

Comment:
Did you mean to remove this changelog?

Reply:
Hmm, no, I didn't. The README is unfortunately part of the same commit as the Paho bump was, I guess. But, that's okay. I'll add back on a second commit to update the README. :-D

Reply:
✅ Fixed